### ISSUE TITLE
job application fails with "Your file couldn't be accessed"

### ISSUE THREAD

**Issue by sswam** [canonical/canonical.com#1264]
## Summary

I tried to apply for the job [Developer Relations Engineer](https://canonical.com/careers/3655681/application).

After taking quite some time to fill in the form, I pressed the **Submit application** button, with the following result:

## Steps to reproduce the behavior

1. Go to the job page [Developer Relations Engineer](https://canonical.com/careers/3655681/application)
2. Fill in the form.
3. Click Submit application
4. See the error "Your file couldn't be accessed"

## Expected behavior

Job application should be submitted.

## Screenshot

![broken](https://github.com/canonical/canonical.com/assets/801764/60e969a9-edd1-48cb-b57c-bc658f13df42)

## Browser/device details

mybrowser.fyi isn't there

- Device: PC
- OS: Debian Linux 12
- Browser: Google Chrome
- Version: 125.0.6422.26

**Comment by britneywwc** [comment]
Hi @sswam, thanks for flagging the issue. I'm having trouble recreating this, could you please confirm if it is still happening on your end?

**Comment by sswam** [comment]
I submitted an application again, and this time it went through okay.

I think the original problem might have been that the file I attempted to upload for my resume was a broken symlink.

That's not something that will happen often, probably never again!  But the user experience wasn't great, as it gave me an error message on a separate page and also lost all the info I had entered.

Thanks and sorry for the user error there.
